src/onboarding/registration/NameAndPicture.tsx

```diff
@@ -135,6 +147,22 @@ function NameAndPicture({ navigation, route }: Props) {
       }
     }
   }
+  const getUsernamePlaceholder = (nameType: OnboardingNameType) => {
+    if (showGuidedOnboarding) {
+      // Firebase trusted-guide onboarding experiment
+      nameType = OnboardingNameType.FirstAndLastOrPseudonym
+    }
```

Comment:
If the `true` side of guided onboarding wins and we roll it out, this would mean that we'll never been using the statsig flag for future experiments right? You'd need a workaround like what I suggested in https://github.com/valora-inc/wallet/pull/2949#discussion_r988144496

Reply:
Right, that is indeed a potential bug.